sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -479,48 +479,55 @@ class BlobAPITest extends APISpec {
         null     | null       | null        | null         | garbageLeaseID
     }
 
-    @Requires({ liveMode() })
-    def "Download file etag lock"() {
-        setup:
-        def file = getRandomFile(1 * 1024 * 1024)
-        bc.uploadFromFile(file.toPath().toString(), true)
-        def outFile = new File(testName + "")
-        if (outFile.exists()) {
-            assert outFile.delete()
-        }
-
-        when:
-        /*
-         Set up a large download in small chunks so it makes a lot of requests. This will give us time to cut in an
-         operation that will change the etag.
-         */
-        def etagConflict = false
-        def bac = new BlobClientBuilder().pipeline(bc.getHttpPipeline()).endpoint(bc.getBlobUrl()).buildAsyncClient()
-            .getBlockBlobAsyncClient()
-        bac.downloadToFileWithResponse(outFile.toPath().toString(), null,
-            new ParallelTransferOptions(1024, null, null), null, null, false)
-            .subscribe({ etagConflict = false }, {
-                if (it instanceof BlobStorageException && ((BlobStorageException) it).getStatusCode() == 412) {
-                    etagConflict = true
-                    return
-                }
-                etagConflict = false
-                throw it
-            })
-
-        sleep(500) // Give some time for the download request to start.
-        bc.getBlockBlobClient().upload(defaultInputStream.get(), defaultDataSize, true)
-
-        sleep(1000) // Allow time for the upload operation
-
-        then:
-        etagConflict
-        !outFile.exists() // We should delete the file we tried to create
-
-        cleanup:
-        file.delete()
-        outFile.delete()
-    }
+//    @Requires({ liveMode() })
+//    def "Download file etag lock"() {
```

Comment:
Need to uncomment this test, even if it continues to be flaky. If it continues to be flaky let's work on stabilizing it in a separate PR.

Reply:
Try to test if tests are able to pass when uncommon this. 
Will revert.